packages/vkui/src/components/SimpleCell/SimpleCell.module.css

```
@@ -23,7 +23,7 @@
   padding-top: 6px;
   padding-bottom: 6px;
   padding-right: 12px;
-  color: var(--vkui--color_icon_accent);
+  color: var(--vkui_internal--icon_color, var(--vkui--color_icon_accent));
```

Красота, только токен стоит более конкретнее сделать

```suggestion
  color: var(--vkui_internal--simple_cell_icons_color, var(--vkui--color_icon_accent));
```